chainercv/datasets/__init__.py

```diff
@@ -15,6 +15,7 @@
 from chainercv.datasets.cub.cub_utils import cub_label_names  # NOQA
 from chainercv.datasets.directory_parsing_label_dataset import directory_parsing_label_names  # NOQA
 from chainercv.datasets.directory_parsing_label_dataset import DirectoryParsingLabelDataset  # NOQA
+from chainercv.datasets.mixup_dataset import MixUpSoftLabelDataset  # NOQA
```

Comment:
Can you change the name of the file to `mixup_soft_label_dataset`?
Also, could you change the name of the test file as well?